espnet2/tts/gradtts/denoiser.py

```diff
@@ -0,0 +1,322 @@
+import math
+
+import torch
+from einops import rearrange
+
+from espnet2.tts.abs_tts import AbsTTS
+
+
+class Mish(AbsTTS):
```

Comment:
```suggestion
class Mish(torch.nn.Module):
```
Please change accordingly.